xinference/core/cache_tracker.py

```diff
@@ -111,3 +113,56 @@ def list_cached_models(self) -> List[Dict[Any, Any]]:
                     cached_models.append(ret)
         cached_models = sorted(cached_models, key=lambda x: x["model_name"])
         return cached_models
+
+    def get_remove_cached_models(
+        self, model_name: str, checked: bool = False
+    ) -> Dict[str, Dict[str, str]]:
+        assert checked == False
+        model_file_location = {}
+        for model, model_versions in self._model_name_to_version_info.items():
+            if model_name.lower() == model.lower():
+                for version_info in model_versions:
+                    if version_info["cache_status"]:
+                        model_file_location = version_info["model_file_location"]
+
+        return {model_name: model_file_location}
+
+    def remove_cached_models(
```

Comment:
`remove` should be excuted by worker, you know, `cache_tracker` was created by `supervisor` so it runs on the device where supervisor locates. if you want to remove path on worker device, `cache_tracker` will tell you `No such file or directory`